ios/RTNAirship.mm

```diff
@@ -117,6 +117,15 @@ + (BOOL)requiresMainQueueSetup {
     [self handleResult:nil error:error resolve:resolve reject:reject];
 }
 
+RCT_REMAP_METHOD(channelEnableChannelCreation,
+                 channelEnableChannelCreation:(RCTPromiseResolveBlock)resolve
+                 reject:(RCTPromiseRejectBlock)reject) {
+    NSError *error;
+    [AirshipReactNative.shared channelEnableChannelCreationAndReturnError:&error];
```

Comment:
I'm not sur about this... I think it's right to add that "AndReturnError" thing but I don't fully understand how it would work lol.
Also, I'm wondering why we still have this class in objective C when we have those methods tagged with @objc in the Swift class?

Reply:
We need the obj-c for c++ bindings with the react plugin. This looks correct to me, thats standard objc bindings for errors